### ISSUE TITLE
load_variables() giving error 

### ISSUE THREAD

**Issue by debipriya** [walkerke/tidycensus#189]
Trying load_variables(2017, "sf1", cache=TRUE) and getting the following error- 
Error in open.connection(x, "rb") : 
  Timeout was reached: Connection timed out after 10016 milliseconds

get_acs() works fine otherwise though.

**Comment by cshancock** [comment]
Similar error when calling the below code. Error reads: "Error in open.connection(x, "rb") : HTTP error 404."

`
load_variables(2010, "sf3", cache = TRUE)
`

**Comment by walkerke** [comment]
You are both running into dataset issues.  There is no such thing as "sf1" for 2017; SF1 refers to summary file 1 which is in the decennial Census.  Instead, you should look for "acs1" for the 1-year ACS or "acs5" for the 5-year ACS.  

Similarly, there is no Summary File 3 in the 2010 Census; this only exists for 1990 and 2000, as the Census Long Form by 2010 (which would have been contained in SF3) had then been replaced by the ACS.

**Comment by corybriar** [comment]
I'm still getting this error in 2020:
`load_variables(1990,"sf3")
Error in open.connection(x, "rb") : HTTP error 404.`
I know that there was a period where sf1 and sf3 were not available for 1990 but it seems like that was years in the past. Unnervingly, I got this code to run a few months ago without a hitch (or so I believe). I've tried refreshing my API key, using variable names from a few months ago, nothing seems to be working.
